components/config-provider/demo/direction.md

```diff
@@ -0,0 +1,172 @@
+---
+order: 2
+title:
+  zh-CN: 方向
+  en-US: Direction
+---
+
+## zh-CN
+
+这里列出了支持 `rtl` 方向的组件，您可以在演示中切换方向。
+
+## en-US
+
+Components which support rtl direction are listed here, you can toggle the direction in the demo.
+
+```jsx
+import { ConfigProvider, Cascader, Radio, Icon } from 'antd';
+
+const cascaderOptions = [
+  {
+    value: 'tehran',
+    label: 'تهران',
+    children: [
+      {
+        value: 'tehran-c',
+        label: 'تهران',
+        children: [
+          {
+            value: 'saadat-abad',
+            label: 'سعادت آیاد',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    value: 'ardabil',
+    label: 'اردبیل',
+    children: [
+      {
+        value: 'ardabil-c',
+        label: 'اردبیل',
+        children: [
+          {
+            value: 'primadar',
+            label: 'پیرمادر',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    value: 'gilan',
+    label: 'گیلان',
+    children: [
+      {
+        value: 'rasht',
+        label: 'رشت',
+        children: [
+          {
+            value: 'district-3',
+            label: 'منطقه ۳',
+          },
+        ],
+      },
+    ],
+  },
+];
+
+class Page extends React.Component {
+  onCascaderChange = value => {
+    console.log(value);
+  };
+
+  cascaderFilter = (inputValue, path) => {
+    return path.some(option => option.label.toLowerCase().indexOf(inputValue.toLowerCase()) > -1);
+  };
+
+  render() {
+    return (
+      <div className="direction-components">
+        <div className="example">
+          <Cascader
+            suffixIcon={<Icon type="smile" />}
+            options={cascaderOptions}
+            onChange={this.onCascaderChange}
+            placeholder="یک مورد انتخاب کنید"
+            popupPlacement={this.props.popupPlacement}
+          />
+          &nbsp;&nbsp;&nbsp;&nbsp; With search:
+          <Cascader
+            suffixIcon={<Icon type="search" />}
+            options={cascaderOptions}
+            onChange={this.onCascaderChange}
+            placeholder="Select an item"
+            popupPlacement={this.props.popupPlacement}
+            showSearch={this.cascaderFilter}
+          />
```

Comment:
Could we provides more demo instead of Cascader only?

Reply:
yes, but for now i just changed styles only for Cascader and each component needs their rtl styles.
I've planned to change each components in separate pull request but I can change more components in this pull request if you want. :)